src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/RazorUpdaterStepExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.UpgradeAssistant.Steps.Razor;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.DotNet.UpgradeAssistant
```

Comment:
is this namespace part of a pattern we've been using and I just didn't notice?

Reply:
The namespace of the service registrations have been at a higher level so that the Program.cs where we tie stuff together does not need to have a bunch of usings